tests/run/forwarder-java-package-private.scala

```diff
@@ -0,0 +1,12 @@
+object MyClassValue extends ClassValue[String] {
```

Comment:
This test isn't guaranteed to keep working since the JDK might add new public methods to ClassValue at any moment. Could you make a test with a custom .java file instead? To test separate compilation the java file needs to end with _1 and the scala file with _2 and both need to be in a subdirectory like in https://github.com/lampepfl/dotty/tree/main/tests/run/java-intersection (checkfile in https://github.com/lampepfl/dotty/blob/main/tests/run/java-intersection.check)